www/starter-redirects.json

```
@@ -59,7 +59,7 @@
   "/gatsby-starter/": "/fabien0102/gatsby-starter/",
   "/gatsby-starter-wordpress/": "/GatsbyCentral/gatsby-starter-wordpress/",
   "/open-crowd-fund/": "/rwieruch/open-crowd-fund/",
-  "/verious-boilerplate/": "/cpinnix/verious-boilerplate/",
+  "/various-boilerplate/": "/cpinnix/various-boilerplate/",
```

```suggestion
  "/verious-boilerplate/": "/cpinnix/verious-boilerplate/",
```
this need to be like that - it's github names